lib/wallet/account.js

```diff
@@ -378,7 +384,7 @@ class Account extends bio.Struct {
         throw new Error(`Bad branch: ${branch}.`);
     }
 
-    this.save(b);
+    await this.save(b);
```

Comment:
What is this code change? I don't want to deviate from master unnecessarily

Reply:
it was not waiting for the result to written out to leveldb, so i thougth it would be a good idea to make it wait write the record and return. but yes it's a deviation. rolling back

Reply:
rolled back